crates/release_plz/src/args/generate_completions.rs

```diff
@@ -0,0 +1,21 @@
+use clap::IntoApp;
+use clap_complete::Shell;
+
+#[derive(clap::Parser, Debug)]
+pub struct GenerateCompletions {
+    /// Shell option
+    #[clap(default_value = "bash")]
+    shell: Shell
+}
+
+impl GenerateCompletions {
+    pub fn print(&self) -> anyhow::Result<()> {
+        clap_complete::generate(
+            self.shell,
+            &mut super::CliArgs::command(),
+            "release-plz",
+            &mut std::io::stdout(),
+        );
+        Ok(())
```

Comment:
```suggestion
    pub fn print(&self) {
        clap_complete::generate(
            self.shell,
            &mut super::CliArgs::command(),
            "release-plz",
            &mut std::io::stdout(),
        );
```
this cannot fail! :)